frontend/src/app/projects/models/backup-settings/create-backup/create-backup.component.html

```diff
@@ -4,70 +4,101 @@
  -->
 
 <div class="dialog">
-  <h1>Create a new backup pipeline</h1>
-  <form [formGroup]="createBackupForm" *ngIf="t4cAndGitModelExists">
-    <div class="flex flex-row flex-wrap items-center">
-      <div class="collab-card">
-        Please select a T4C model:
-        <mat-selection-list formControlName="t4cmodel" [multiple]="false">
-          <mat-list-option
-            *ngFor="let t4cModel of t4cModelService.t4cModels$ | async"
-            [value]="t4cModel.id"
+  <h2 class="mb-5 text-xl font-medium">Create a new backup pipeline</h2>
+  @if (
+    (this.gitModelService.gitModels$ | async) === undefined ||
+    (this.t4cModelService.t4cModels$ | async) === undefined
+  ) {
+    <ngx-skeleton-loader
+      appearance="circle"
+      [theme]="{
+        'border-radius': '5px',
+        height: '100px',
+        width: '500px',
+        border: '1px solid white',
+      }"
+    ></ngx-skeleton-loader>
+  } @else if (
+    (this.gitModelService.gitModels$ | async)?.length &&
+    (this.t4cModelService.t4cModels$ | async)?.length
+  ) {
+    <form [formGroup]="createBackupForm">
+      <div class="flex flex-row flex-wrap items-center gap-2">
+        <div class="collab-card">
+          Please select a T4C model. <br />
+          The project name has to match exactly.
+          <mat-selection-list
+            class="!flex flex-col gap-2"
+            formControlName="t4cmodel"
+            [multiple]="false"
           >
-            <div mat-line>TeamForCapella model '{{ t4cModel.name }}'</div>
-            <div mat-line>
-              repository '{{ t4cModel.repository.name }}', instance '{{
-                t4cModel.repository.instance.name
-              }}'
-            </div>
-          </mat-list-option>
-        </mat-selection-list>
-      </div>
-      <mat-icon>arrow_right_alt</mat-icon>
-      <div class="collab-card">
-        Please select a Git model:
-        <mat-selection-list formControlName="gitmodel" [multiple]="false">
-          <mat-list-option
-            *ngFor="let gitModel of gitModelService.gitModels$ | async"
-            [value]="gitModel.id"
+            @for (
+              t4cModel of t4cModelService.t4cModels$ | async;
+              track t4cModel.id
+            ) {
+              <mat-list-option class="!h-auto !rounded" [value]="t4cModel.id">
+                <div mat-line>
+                  TeamForCapella link (No. {{ t4cModel.id }}) <br />
+                  <b>Project:</b> {{ t4cModel.name }}
+                </div>
+                <div mat-line>
+                  <b>Repository:</b> {{ t4cModel.repository.name }} <br />
+                  <b>Instance:</b> {{ t4cModel.repository.instance.name }}
+                </div>
+              </mat-list-option>
+            }
+          </mat-selection-list>
+        </div>
+        <mat-icon>arrow_forward</mat-icon>
+        <div class="collab-card">
+          Please select a Git model.
```

Comment:
I think this would look better if it was a bold heading with just "Git Model".

Reply:
Done